molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/platform/OneToManyIT.java

```diff
@@ -526,8 +505,14 @@ private void deleteBooksThenAuthors(int testCase) {
   @Autowired private PermissionService testPermissionService;
 
   private void populateUserPermissions() {
-    testPermissionService.grant(
-        getPermissionMap(), createUserSid(requireNonNull(getCurrentUsername())));
+    Sid sid = SidUtils.createUserSid(USERNAME);
```

Comment:
Do you need to clean up permissions here similar to PlatformIT?

Reply:
No since this test class shares the "setPermissions" in the "before" preventing the problems that occur in PlatformIt (where permissions are set in every test method, partially overlapping)